tools/GraphClust/AggregateAlignments/graphclust_aggregate.xml

```diff
@@ -0,0 +1,60 @@
+<tool id="graphclust_aggregate" name="Aggregate and filter alignment metrics" version="0.1">
```

Comment:
btw. is this one missing a shed.yml?

Why is the tool_id not `graphclust_aggregate_alignments?